src/System.Net.NetworkInformation/tests/FunctionalTests/project.json

```diff
@@ -0,0 +1,11 @@
+{
+  "dependencies": {
+    "System.Net.Primitives": "4.0.10-beta-*",
+
+    "xunit": "2.1.0-beta3-*",
+    "xunit.netcore.extensions": "1.0.0-prerelease-*"
```

Comment:
@ianhays just updated the test projects to use a more recent and stable xunit.  We'll want to snap to that versioning before checking this in.